### DISCUSSION TITLE
Problem activating the github pilot

### DISCUSSION THREAD

**Discussion by NanciRamirez** [microsoft/SATechnicalOnboarding#72] [Questions]
Hi everyone, I'm having a hard time getting started with my technical onboarding, I was currently going through the step of creating the repository but I get stuck in a step where the co-pilot GitHub asks me, and to activate it I had to buy a monthly plan which I entered my debit card and it only read the data and said that the website did not exist,   So I don't understand?

**Comment by Sameer1103** [comment]
I think there is no need to buy a plan for GitHub Copilot. We can access the GitHub student developer pack by validating our student identity card

**Reply by RimmalapudiRajesh** [reply]
i am also facing the same problem

**Reply by Cankrs2** [reply]
Did you solve the problem?
